server/services/interventionsService.ts

```diff
@@ -125,6 +125,35 @@ export interface InterventionsFilterParams {
   maximumAge?: number
 }
 
+export interface DraftActionPlan extends WithNullableValues<ActionPlanFields> {
+  id: string
+}
+
+interface ActionPlanFields {
+  referralId: string
+  activities: Activity[] | null
+  numberOfSessions: number | null
+}
+
+interface Activity {
+  id: string
+  desiredOutcomeId: string
```

Comment:
For submitting an activity there is a desiredOutcomeId
Responses contain the full desiredOutcome element with id and desc